app/components/header/extra_navigation_component.html.erb

```diff
@@ -1,12 +1,17 @@
 <%= tag.div(class: classes, data: { controller: "searchbox", "searchbox-search-input-id-value" => search_input_id }, role: "navigation") do %>
   <ul class="extra-navigation__list">
-    <li class="extra-navigation__link extra-navigation__mail">
-      <%= link_to("/mailinglist/signup/name", class: "link--black") do %>
-      Register your interest<!--
-   --><span class="icon icon-mail" aria-hidden="true"></span>
-      <span class="icon icon-mail-white" aria-hidden="true"></span>
-      <% end %>
-    </li>
+    <span class="extra-navigation__flex">
+      <li class="extra-navigation__link extra-navigation__mail">
+        <%= link_to("/mailinglist/signup/name", class: "link--black") do %>
+        Sign up for emails<span class="icon icon-mail" aria-hidden="true"></span><span class="icon icon-mail-white" aria-hidden="true"></span>
+        <% end %>
+      </li>
+      <li class="extra-navigation__link extra-navigation__calendar">
+        <%= link_to("/events", class: "link--black") do %>
```

Comment:
@garethwilliamsdesign @philspray @EllieNodder 

I think we should probably point this link to `/events/about-get-into-teaching-events` rather than the main listing page, as we are really wanting to promote GIT events first and foremost, particularly given this term's events have just been launched?

We do link back to other

Reply:
However, if we do that, we will need to consider the link text on either this bonus bar or on the home page events block, as then we'd have two instances of `Find an event` links going to different locations - not good in terms of accessibility